src/formatters/index.ts

```diff
@@ -2,12 +2,14 @@ import type { Props } from '../Flash';
 
 export type Formatter = (value: Props['value']) => string;
 
-type Formatters = {
+export type Formatters = {
```

Comment:
Does this get used outside of this file? I don't see it being imported anywhere.